### PR TITLE
Fix axios cache issue

### PR DESCRIPTION
**Pull request by pkhien95**
Add timestamp into GET request so the schema json will not be cached

### CODE REVIEW

**Comment by nick-verida** [comment]
@pkhien95 what is the status of this change?

File: packages/client-ts/src/context/schema.ts [hunk 1]
```diff
@@ -136,7 +136,10 @@ export default class Schema {
 
         const path = await this.getPath();
         const fileData = await axios.get(path, {
-            responseType: 'json'
+            responseType: 'json',
+            params: {
+                t: new Date().getTime()
```
[the remark this answers is not in the document]
Comment (tahpot) [reply packages/client-ts/src/context/schema.ts]: @pkhien95 I've changed my mind here. Feel free to use this code, but it needs to be on internal testing builds only -- not on production.

[the remark this answers is not in the document]
Comment (nick-verida) [reply packages/client-ts/src/context/schema.ts]: @tahpot @pkhien95 there's probably not much point having any caching for test builds (no load) so probably should disable the cache completely.

[the remark this answers is not in the document]
Comment (tahpot) [reply packages/client-ts/src/context/schema.ts]: @nick-verida The issue is the phone automatically caches... so this code forces a timestamp added to the URL which prevents the caching.

**Comment by pkhien95** [comment]
> @pkhien95 I've changed my mind here. Feel free to use this code, but it needs to be on internal testing builds only -- not on production.

To detect environments (production / internal) I will need to pass a variable from the vault to the initial process of client-ts, @tahpot @nick-verida is that outside of the scope of this PR?

**Comment by nick-verida** [comment]
@tahpot can I close this PR without merging?

**Comment by tahpot** [comment]
Yes, closing now.

**Pull request closed without merging**